tools/generator/AnDOGenerator.py

```diff
@@ -142,7 +143,7 @@ def get_all_folder_paths(self):
         assert not check_Path(combined_paths, verbose=False)[0], \
             'Error in AnDO path generation. Generated paths are not consistent with AnDO ' \
             'specifications'
-
+        print(paths)
```

Comment:
same here: I guess this is still a leftover from debugging? Shouldn't stay in the codebase.

Reply:
deleted